charts/rucio-server/templates/servicemonitor.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if gt .Values.replicaCount 0.0 -}}
-{{- if .Values.monitoring.enabled }}
+{{- if and .Values.monitoring.enabled .Values.monitoring.serviceMonitorEnabled }}
```

Comment:
It seems to me that to not break existing installations, you should set monitoring.serviceMonitorEnabled to true in values.yaml. And then leave a little comment, since this is confusing, that monitoring also has to be enabled (it's false by default).